streams/writable-streams/close.js

```diff
@@ -60,6 +60,22 @@ promise_test(t => {
   return writer.close().then(() => promise_rejects(t, passedError, writer.closed, 'closed should stay rejected'));
 }, 'when sink calls error synchronously while closing, the stream should become errored');
 
+promise_test(t => {
+  const ws = new WritableStream({
+    close() {
+      throw error1;
+    }
+  });
+
+  const writer = ws.getWriter();
+
+  return Promise.all([
+    writer.write('y'),
+    promise_rejects(t, error1, writer.close(), 'close() must reject with the error'),
+    promise_rejects(t, error1, writer.closed, 'closed must reject with the error')
+  ]);
+}, 'when the sink throws and a write is queued up, the stream should become errored');
```

Comment:
Yes, sure, but it becomes errored during the close. I'll add "during the close"?